test/types/tsconfig.json

```
@@ -0,0 +1,28 @@
+{
+	"compilerOptions": {
+		"target": "es2019" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
+		"lib": [
+			"es2019.object",
+		],
+		"noEmit": true, /* Do not emit outputs. */
+		/* Strict Type-Checking Options */
+		"strict": true /* Enable all strict type-checking options. */,
+		"noImplicitAny": true, /* Raise error on expressions and declarations with an implied 'any' type. */
+		"strictNullChecks": true, /* Enable strict null checks. */
+		"strictFunctionTypes": true, /* Enable strict checking of function types. */
+		"strictBindCallApply": true, /* Enable strict 'bind', 'call', and 'apply' methods on functions. */
+		"strictPropertyInitialization": true, /* Enable strict checking of property initialization in classes. */
+		"noImplicitThis": true, /* Raise error on 'this' expressions with an implied 'any' type. */
+		"alwaysStrict": true /* Parse in strict mode and emit "use strict" for each source file. */,
```

`"strict": true` turns in the options that are then listed below, it can just be:

```suggestion
		"strict": true /* Enable all strict type-checking options. */,
```